release-process.md

```diff
@@ -1,12 +1,17 @@
 # Release process 
 
 This document includes:
- - how to update remix.ethereum.org.
- - how to update remix-alpha.ethereum.org.
- - how to release remix IDE.
+ - how to publish remix libs to NPM
+ - how to update remix.ethereum.org
+ - how to update remix-alpha.ethereum.org
+ - how to release remix IDE
 
 ## Remix libs release
- - TODO
+ - git fetch origin master
```

Comment:
perhaps we should (in another PR and md maybe) define what's the process of releasing in github.